### PR TITLE
Make from page statistics more accessible

### PR DESCRIPTION
**Pull request by joelverhagen**
Done by making the `<div>` have an image role while providing a descriptive `aria-label`.

Also, removed .png circuit board (was complicating high contrast work) and hid the circuit board when SVG is not supported.

/cc @microsoftsam

Address https://github.com/NuGet/NuGetGallery/issues/4102.

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 1]
```diff
@@ -198,6 +201,66 @@
...
 
+            @if (Model.Dependencies.DependencySets == null)
+            {
+                if ((Model.IsOwner(User) || User.IsAdministrator()))
+                {
+                    <h2>Dependencies</h2>
+                    <p>
+                        An error occurred processing dependencies.
+                        Your package can still be downloaded and installed, but dependencies cannot be shown.
+                        Please @Html.ActionLink("Contact Support", actionName: "ReportMyPackage", controllerName: "Packages", routeValues: new { id = Model.Id, version = Model.Version }, htmlAttributes: null) if you have questions.
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: nit: `if you have questions` -> `if you have any questions`

File: src/NuGetGallery/Views/Pages/Home.cshtml [hunk 2]
```diff
@@ -20,18 +20,21 @@
                 </div>
             </div>            
         </div>
-        <div class="circuit-board hidden-xs">
+        <div class="circuit-board hidden-xs" role="img" data-bind="attr: { 'aria-label': label }">
             <div class="circuit-board-label circuit-board-pd triangle triangle-left">
-                <h3 data-bind="animateNumber: packageDownloads">0</h3>
-                <h5>package downloads</h5>
+                <span class="ms-font-xxl" data-bind="animateNumber: packageDownloads">0</span>
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Pages/Home.cshtml]: Should we start out with a value that's not 0? One that signifies that the stats haven't loaded yet?

(Maybe not in the scope of this PR...)

Comment (joelverhagen) [inline src/NuGetGallery/Views/Pages/Home.cshtml]: yep, out of scope

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml (hunk 1 above)
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Fixed.

**Pull request merged**